Saqo/databases/29-04-24/hospital_part_2.sql

```diff
@@ -0,0 +1,22 @@
+select h.address, h.name, d.name, dc.name, p.name from
+Hospital h inner join Department d on h.id = d.hospital_id
+inner join Doctor dc on dc.id = d.id
+inner join Patient p on p.id = d.hospital_id
```

Comment:
`p.id = d.hospital_id` is an incorrect comparison, as they are not the same column

Reply:
Done ✅